x/token/keeper/querier.go

```diff
@@ -1,21 +1,27 @@
 package keeper
 
 import (
-	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/Decentr-net/decentr/x/token/types"
 	abci "github.com/tendermint/tendermint/abci/types"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
-
-	"github.com/Decentr-net/decentr/x/utils"
 )
 
 const (
 	QueryBalance = "balance"
+	QueryPool    = "pool"
```

Comment:
Is there a way to query PDV reward distribution history of the specific account?

Reply:
Oops